plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -174,6 +175,8 @@
     @Parameter(property = "fabric8.build.strategy")
     private OpenShiftBuildStrategy buildStrategy = OpenShiftBuildStrategy.s2i;
 
+    @Parameter(property = "fabric8.build.switchToDeployment", defaultValue = "true")
```

Comment:
Would it be better to keep the default preference to `false`, so that existing users will not get surprising behavior :)
@jstrachan WDYT?